en/api/clapi.md

```diff
@@ -3569,18 +3569,26 @@ Columns are the following:
 
 In order to add an instance you use the **ADD** action:
 
-``` shell
-centreon -u admin -p centreon -o INSTANCE -a add -v "Poller test;10.30.2.55;22;ZMQ;5556"
-```
+- For a ZMQ poller:
+
+    ``` shell
+    centreon -u admin -p centreon -o INSTANCE -a add -v "Poller test;10.30.2.55;22;ZMQ;5556"
+    ```
+
+- For an SSH poller:
+
+    ``` shell
+    centreon -u admin -p centreon -o INSTANCE -a add -v "Poller test;10.30.2.55;22;2;22"
```

Comment:
For ZMQ you use the string, but SSH the id. I think it's a typo if I understand the array below.